README.md

```diff
@@ -64,9 +64,38 @@ Run it as
 See `jsonrcpproxy.py --help` for more options.
 
 
+## geth
+
+This section describes how to run geth node with the wagon backend.
+
+1. Get the code from the PR
+
+	```sh
+	go get github.com/ethereum/go-ethereum
+	cd $GOROOT/src/github.com/ethereum/go-ethereum
+	git remote add gballet git@github.com:gballet/go-ethereum.git
+	git fetch gballet add-ewasm
+	git checkout add-ewasm
```

Comment:
If you add `gballet` as a remote why not just check it out from there? Also how about pushing a maintained version of it to https://github.com/ewasm/go-ethereum ?

Reply:
because of the `$GOROOT` crap and because I would have to maintain it.